packages/app/src/runner/SpecRunnerContainerOpenMode.vue

```diff
@@ -31,4 +29,14 @@ const specs = computed(() => {
 })
 
 watchSpec(specs)
+
+watchEffect(() => {
+  const specPath = router.currentRoute.value.query.file
```

Comment:
Can use `useRoute`

```js
const route = useRoute()

watchEffect(() => {
  if (route.query.file && !specStore.activeSpec) {
    router.push({ name: 'Specs', params: {
      unrunnable: route.query.file as string,
    } })
  }
})
```

Reply:
Since we already have the `router` it seemed simpler to just grab the current route from it, since this is only time we touch it.